src/components/Inputs/Input/Input.module.scss

```
@@ -0,0 +1,49 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+.label {
+  font-size: 1.125rem;
+  font-weight: 500;
+
+  .required {
+    color: $color-violet-5534da;
+  }
+}
+
+.inputContainer {
+  display: inline-flex;
+  border: 0.0625rem solid $color-gray-d9d9d9;
+  border-radius: 0.5rem;
+  width: min(100%, 32.5rem);
+  height: 3rem;
+  padding: 0 1rem;
+  gap: 0.5rem;
+
+  &.error {
+    border-color: $color-red-d6173a;
+  }
+
+  &:focus-within {
```

```suggestion
  &:focus-within {
```

전 인풋 테두리로 컨트롤 했었는데 이런 것도 있었군요👍⭐️👍⭐️👍⭐️👍⭐️